clusterloader2/testing/load/experimental-config.yaml

```diff
@@ -113,7 +113,7 @@ steps:
     Method: APIResponsivenessPrometheus
     Params:
       action: start
-  - Identifier: LoadPodStartupLatency
+  - Identifier: SecondPodStartupLatency
```

Comment:
I don't like "second", let's make the name more descriptive for people who are not familiar how load test is structured. 
It will be better even for us, I'm quite sure that we'll forget what "second" refers to after some time.

What about LoadPhasePodStartupLatency or CreatePhasePodStartupLatency or something like that?

Reply:
Changed to `CreatePhasePodStartupLatency`. `Load...` creates a confusion with a name of a whole test.